### PR TITLE
handle invalid issues, retry after removing invalid IDs

### PR DESCRIPTION
**Pull request by hoxhaeris**

### CODE REVIEW

File: pkg/release-controller/release_info.go [hunk 1]
```diff
@@ -228,7 +228,7 @@ func (r *ExecReleaseInfo) ChangeLog(from, to string, isJson bool) (string, error
 	if err != nil {
 		return "", fmt.Errorf("Failed to find `oc` executable: %w", err)
 	}
-	args := []string{"adm", "release", "info", "--changelog=/tmp/git/", from, to}
+	args := []string{"adm", "release", "info", "--changelog=/home/ehoxha/git/", from, to}
```
Comment (bradmwilliams) [inline pkg/release-controller/release_info.go:231]: This can't be

**Review by bradmwilliams — CHANGES_REQUESTED** [review]

File: pkg/release-controller/release_info.go (hunk 1 above)
Comment (hoxhaeris) [reply pkg/release-controller/release_info.go:231]: :facepalm: , thanks

**Review by AlexNPavel — APPROVED** [review]

**Pull request merged**